nevergrad/optimization/test_optimizerlib.py

```diff
@@ -523,7 +523,7 @@ def _ellips(x: np.ndarray) -> float:
     "penalization,expected,as_layer",
     [
         (False, [1.005573e00, 3.965783e-04], False),
-        (True, [1.000001, 0.002842], False),
+        (True, [0.999924, -0.111009], False),
```

Comment:
notice that it deteriorates the results, because it runs only 5 trials now instead of around 50 at the beginning